styles.css

```diff
@@ -1,33 +1,91 @@
 
 .container {
+    background-color: white;
+    color: white;
+    font-size: larger;
+    display: grid;
+    column-gap: 16px;
+    row-gap: 16px;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 210px 100px auto 50px;
+    grid-template-areas:
+            "item-a item-c item-c"
+            "item-b item-c item-c"
+            "item-b item-e item-d"
+            "item-b item-e item-leeg"
```

Comment:
een punt zo ook werken als er op die row geen content zou moeten komen. Een lege <div> of item zou dan ook niet nodig zijn